packages/smooth_app/test/pages/user_preferences_page_test.dart

```diff
@@ -0,0 +1,62 @@
+import 'dart:io';
+
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:openfoodfacts/personalized_search/product_preferences_selection.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:smooth_app/data_models/product_preferences.dart';
+import 'package:smooth_app/data_models/user_preferences.dart';
+import 'package:smooth_app/pages/user_preferences_page.dart';
+import 'package:smooth_app/themes/theme_provider.dart';
+
+import '../utils/goldens.dart';
+import '../utils/mocks.dart';
+
+void main() {
+  HttpOverrides.global = MockHttpOverrides();
+
+  group('UserPreferencesPage looks as expected', () {
+    for (final String color in <String>[ 'blue', 'brown', 'green' ]) {
```

Comment:
Should the code be formatted with `flutter format`?

Reply:
Oh definitely! I did not realize that was the preferred format here. :)